code/modules/mob/login.dm

```diff
@@ -103,3 +103,34 @@
 		return client.holder.auto_deadmin()
 	if(job)
 		return SSjob.handle_auto_deadmin_roles(client, job)
+
+
+
+/mob/attack_ghost(mob/user)
```

Comment:
/mob/living would be a much better candidate to have this in. I see no reason in any situation a ghost or anything other than a living mob would need to be controlable

Reply:
Alright. Do you know where/how I can set mobs to appear into the mob list ghost menu?

Reply:
The spawners menu?